webapp/js/components/Utils.js

```diff
@@ -23,3 +23,35 @@ export function T(message) {
     return msg
 }
 
+export function isHTTPError(response) {
+    // Check for an HTTP error
+    if ((response.statusCode >= 300) && (response.statusCode != 400)) {
```

Comment:
I've found that the function is not used. It's imported in AccountForm.js, but has been replaced with parseError. I'll merge for now and remove it in the next PR, which is in progress.